### PR TITLE
Ttt lite refactored

### PR DESCRIPTION
**Pull request by Sillhouette**

### CODE REVIEW

File: test/ticTacToeLiteTest.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        ttt.enableLite()
+  })
+
+  describe("#addMoveToHistory", function() {
+
+    it("can add a move to the history", function() {
+        expected = 4
+
+        tttLite.addMoveToHistory(expected)
+        actual = tttLite.getRemovalIndex()
```
Comment (tpetersen0308) [inline test/ticTacToeLiteTest.js]: Can you think of any drawbacks to making this test dependent on methods that are not themselves under test in this scope?

File: ticTacToe.js [hunk 2]
```diff
@@ -808,35 +544,112 @@ function newGameComputerFirst() {
...
+        }
+
+        if(numPlayers === 1){
+            performMove(index, "X", 1)
+            if(!isGameOver()) {
+                performMove(computerMove(), "O", 2)
+            }
+        }
+    }
+}
```
Comment (tpetersen0308) [inline ticTacToe.js]: A lot of nesting going on here. Can you use return statements to control flow here and reduce that perhaps?

Comment (Sillhouette) [reply ticTacToe.js]: Addressed in acec4ff

File: test/ticTacToeLiteTest.js (hunk 1 above)
Comment (Sillhouette) [reply test/ticTacToeLiteTest.js]: Addressed in 6b78ebf

**Pull request merged**